Missing spacing between nested form fields

### Describe the Bug

The spacing between nested form fields is missing. See screenshot.

### Steps to Reproduce the Bug or Issue

1. Click on a "from" node in the visual editor.
2. Compare the spacing of top-level form fields and nested fields. You should notice the missing spacing between the nested form fields.

### Screenshots or Videos

![from-configuration](https://github.com/KaotoIO/kaoto-next/assets/1910095/e8bdfcb8-66b2-4168-a858-7d0183843e64)


### Platform

- OS: Linux
- Browser: Chrome
- Version: 120.0.6099.224

@lordrip 
I went through the repo and might need your help in understanding a few things:
1. How can I test changes locally after making changes to the uniforms-patternfly repo. Do we do this by linking the 2 projects?
2. I tried the npm link process but unable to with the following error:
```
npm link @kaoto-next/uniforms-patternfly
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*

```
3. While trying to link with yarn, I directly ran the command `yarn link ../../../uniforms-patternfly` on the UI workspace providing the relative path to the other repo. I am getting the following error:
`Cannot link @kaoto-next/uniforms-patternfly into @kaoto-next/ui$wsroot$@workspace:packages/ui dependency sass@npm:1.70.0 conflicts with parent dependency sass@npm:1.66.1`

--- a/packages/ui/src/components/Visualization/Canvas/CanvasForm.scss
+++ b/packages/ui/src/components/Visualization/Canvas/CanvasForm.scss
@@ -1,16 +1,3 @@
-/*
-  This is required since there's a div between the form
-  element and its children that breaks the following styles
-    - display: grid
-    - gap: var(--pf-c-form--GridGap)
-  from @patternfly and making the fields very close to
-  each other
-*/
-form[data-testid='autoform'] > div {
-  display: inherit;
-  gap: inherit;
-}
-
 .canvas-form {
   overflow: auto;
 }

